content/python/hardware_components.py.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Hardware components and versions"
+description: "Example on how to get hardware components and versions"
+date: "2021-04-27"
+classes: 
+    - "SoftLayer_Account"
+    - "SoftLayer_Hardware_Server"
+tags:
+    - "hardware"
+---
+
+This python script contains the following:
+
+1.- Getting hardware information.
+
+2.- Getting the hardware components.
+
+3.- Getting hardware components firmware version information.
+
+4.- Updating the the firmware on components.
+
+```python
+
+import SoftLayer
+from prettytable import PrettyTable
+
+
+class HardwareComponents():
+
+    def __init__(self):
+
+        self.client = SoftLayer.create_client_from_env()
+
+    def hardware_information(self, server_id):
```

Comment:
changes `hardware_detail(self, identifier)`

Reply:
is possible add a little description

Reply:
Done.